### DISCUSSION TITLE
Can dp ignore some of the data in atom_ener.npy?

### DISCUSSION THREAD

**Discussion by cjxxjc729** [deepmodeling/deepmd-kit#1835] [General]
Dear all, 

While training, I use atom_ener.npy as the only training input data ( I have already set start and limit _pref_e, pref_f, pref_v to be zero, and start_pref_ae and limit_pref_ae to be one).  

But I do not want to learn all of the data from atom_ener.npy.  Can I make dp to selectively learn only part of the data?  The remaining unlearned data is set only as the surroundings.

I want to do so because I have already known some of the atom_ener.npy data are inaccurate

**Comment by wanghan-iapcm** [comment]
Removing the frames from the datasets will work.

**Reply by cjxxjc729** [reply]
Thank wanghan for the reply. But my actual meaning is like, assume I got a system of 70 atoms, I only want 4 atoms' atomic energy information.  For the left 66 atoms, I don't want DP to study their information, but they can still be set as the surroundings of the 4 atoms. Is this possible in DP?
